### PR TITLE
Fixes colab links for lightning in 2steps documentation

### LINKED ISSUE #8684: Links to Colab in "Lightning in 2 steps" are dead
## 📚 Documentation

Under "Examples of systems are:" all the Colab links are dead.

### PR DESCRIPTION
**Pull request by seanbenhur**
## What does this PR do?

Fixes #8684 

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/starter/new-project.rst [hunk 1]
```diff
@@ -120,10 +120,10 @@ A :doc:`lightning module <../common/lightning_module>` defines a *system* not a
...
-- `DQN <https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/notebooks/08-Domain-specific-demos.ipynb>`_
-- `GAN <https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/notebooks/03-basic-gan.ipynb>`_
-- `Image classifier <https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/notebooks/01-mnist-hello-world.ipynb>`_
+- `BERT <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/6100885854c803458886c731cddd6bd67498c0a1/.notebooks/lightning_examples/text-transformers.ipynb>`_
+- `DQN <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/6100885854c803458886c731cddd6bd67498c0a1/.notebooks/lightning_examples/reinforce-learning-DQN.ipynb>`_
+- `GAN <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/6100885854c803458886c731cddd6bd67498c0a1/.notebooks/lightning_examples/basic-gan.ipynb>`_
+- `Image classifier <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/6100885854c803458886c731cddd6bd67498c0a1/.notebooks/lightning_examples/mnist-hello-world.ipynb>`_
```
Comment (rohitgr7) [inline docs/source/starter/new-project.rst:126]:
```suggestion
- `BERT <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/publication/.notebooks/lightning_examples/text-transformers.ipynb>`_
- `DQN <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/publication/.notebooks/lightning_examples/reinforce-learning-DQN.ipynb>`_
- `GAN <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/publication/.notebooks/lightning_examples/basic-gan.ipynb>`_
- `Image classifier <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/publication/.notebooks/lightning_examples/mnist-hello-world.ipynb>`_
```

all are available under `publication` branch.

File: docs/source/starter/new-project.rst [hunk 2]
```diff
@@ -120,10 +120,10 @@ A :doc:`lightning module <../common/lightning_module>` defines a *system* not a
...
 - Seq2seq
 - `SimCLR <https://github.com/PyTorchLightning/lightning-bolts/blob/master/pl_bolts/models/self_supervised/simclr/simclr_module.py>`_
 - `VAE <https://github.com/PyTorchLightning/lightning-bolts/blob/master/pl_bolts/models/autoencoders/basic_vae/basic_vae_module.py>`_
```
Comment (rohitgr7) [inline docs/source/starter/new-project.rst:129]:
```suggestion
- `VAE <https://github.com/PyTorchLightning/lightning-bolts/blob/master/pl_bolts/models/autoencoders/basic_vae/basic_vae_module.py>`_
- [and a lot more](https://github.com/PyTorchLightning/lightning-tutorials/tree/publication/.notebooks/lightning_examples)
```

maybe :)

**Review by ethanwharris — APPROVED** [review]
LGTM :smiley:

**Review by awaelchli — APPROVED** [review]

File: docs/source/starter/new-project.rst [hunk 3]
```diff
@@ -120,13 +120,14 @@ A :doc:`lightning module <../common/lightning_module>` defines a *system* not a
...
+- `BERT <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/publication/.notebooks/lightning_examples/text-transformers.ipynb>`_
+- `DQN <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/publication/.notebooks/lightning_examples/reinforce-learning-DQN.ipynb>`_
+- `GAN <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/publication/.notebooks/lightning_examples/basic-gan.ipynb>`_
+- `Image classifier <https://colab.research.google.com/github/PyTorchLightning/lightning-tutorials/blob/publication/.notebooks/lightning_examples/mnist-hello-world.ipynb>`_
 - Seq2seq
 - `SimCLR <https://github.com/PyTorchLightning/lightning-bolts/blob/master/pl_bolts/models/self_supervised/simclr/simclr_module.py>`_
 - `VAE <https://github.com/PyTorchLightning/lightning-bolts/blob/master/pl_bolts/models/autoencoders/basic_vae/basic_vae_module.py>`_
+- [and a lot more](https://github.com/PyTorchLightning/lightning-tutorials/tree/publication/.notebooks/lightning_examples)
```
Comment (rohitgr7) [inline docs/source/starter/new-project.rst:130]:
```suggestion
- Seq2seq
- `SimCLR <https://github.com/PyTorchLightning/lightning-bolts/blob/master/pl_bolts/models/self_supervised/simclr/simclr_module.py>`_
- `VAE <https://github.com/PyTorchLightning/lightning-bolts/blob/master/pl_bolts/models/autoencoders/basic_vae/basic_vae_module.py>`_
- `and a lot more <https://github.com/PyTorchLightning/lightning-tutorials/tree/publication/.notebooks/lightning_examples>`_
```

**Review by rohitgr7 — APPROVED** [review]

**Pull request merged**